client/my-sites/upgrades/checkout-thank-you/style.scss

```diff
@@ -145,31 +145,76 @@
 
 // Footer Subcomponent styles
 .checkout-thank-you__footer {
-	color: $gray;
+	padding: 32px;
+	text-align: left;
+
+	@include breakpoint( ">660px" ) {
+		padding-bottom: 48px;
+		padding-left: 24px;
+		padding-right: 24px;
+		padding-top: 48px;
```

Comment:
That's true but it's also much more easier to understand when each margin is explicitly stated. Personally speaking I always forget what these numbers represent in the format you suggest :).